apps/sms/services/js/moz_settings/moz_settings_client.js

```diff
@@ -0,0 +1,83 @@
+/* global bridge,
+          BroadcastChannel */
+
+(function(exports) {
+'use strict';
+
+/**
+ * Name of the service that is responsible for settings functionality.
+ * @type {string}
+ */
+const SERVICE_NAME = 'moz-settings-shim';
+
+/**
+ * Disable default client timeout from bridge by setting the timeout to false.
+ * @type {number|boolean}
+ */
+const TIMEOUT = false;
+
+const MMS_SIZE_LIMIT_KEY = 'dom.mms.operatorSizeLimitation';
+const SMS_MAX_CONCAT_KEY = 'operatorResource.sms.maxConcat';
+
+// We set the default maximum concatenated number of our SMS app to 10
+// based on:
+// https://bugzilla.mozilla.org/show_bug.cgi?id=813686#c0
+const MAX_CONCATENATED_MESSAGES = 10;
+// Default mms message size limitation is 295K
+const MMS_SIZE_LIMITATION = 295 * 1024;
+// we evaluate to 5KB the size overhead of wrapping a payload in a MMS
+const MMS_SIZE_OVERHEAD = 5 * 1024;
+
+/**
+ * Reference to active bridge client instance.
+ *
+ * @type {Client}
+ */
+var client;
+
+var MozSettingsClient = {
+  /**
+   * Initializes connection to mozSettingsShim hosted in iframe.
```

Comment:
__optional nit(:bike:):__ with this new planned changes about LittleBrowser and alike maybe we shouldn't be that specific and say ```hosted in iframe```, maybe just omit this or name ```hosted in a different compartment``` as you wish, but it's optional and doesn't make any difference anyway.